web/static/js/components/simulation/VoiceWindow.jsx

```diff
@@ -64,7 +64,9 @@ const VoiceWindow = translate()(class extends Component<VoiceWindowProps, VoiceW
       this.audio.pause()
       this.audio.src = audioURL(ivr)
       this.audio.onended = () => { this.play() }
-      this.audio.play()
+
+      const promise = this.audio.play()
+      promise && promise.catch(() => {}) // silence async play errors
```

Comment:
Should we do something like in the example [here]( https://developers.google.com/web/updates/2017/06/play-request-was-interrupted)
```
const promise = this.audio.play()
if (promise !== undefined) {
    promise.then(_ => {
      // We can now safely pause video...
      this.audio.pause();
    })
    .catch(error => {
      // Auto-play was prevented
      // Show paused UI.
    });
  }
```

And remove line 64 `this.audio.pause()` that I think is part of the problem

Reply:
I wasn't sure, but we can indeed delay the handling of the `play()` promise to the next call to `VoiceWindow.play()` and we can silence the error at this moment to pause/load/play the next IVR audio prompt, with a fallback for the first audio _and_ older browsers that don't return a promise.